draft-ietf-acme-acme.md

```diff
@@ -955,18 +948,17 @@ key but not the corresponding registration URI to recover the registration URI.
 
 If the server wishes to present the client with terms under which the ACME
 service is to be used, it MUST indicate the URI where such terms can be accessed
-in a Link header with link relation "terms-of-service".  As noted above, the
-client may indicate its agreement with these terms by updating its registration
-to include the "agreement" field, with the terms URI as its value.  When these
-terms change in a way that requires an agreement update, the server MUST
-use a different URI in the Link header.
+in the "terms-of-service" subfield of the "meta" field in the directory object,
+and the server MUST reject new-registration requests that do not contain
+"terms-of-service-agreed": true. Note that the "terms-of-service-agreed" field
+is not echoed in the resulting registration object, because it is a one-time
+field.
```

Comment:
I'm +1 on echoing it back - I don't think it would upset clients and is less special-casing.